app/policies/request_for_comment_policy.rb

```diff
@@ -41,6 +41,16 @@ def rfcs_with_my_comments?
     everyone
   end
 
+  def report?
+    report_receiver_configured? && show? && !author?
+  end
+
+  private
+
+  def report_receiver_configured?
+    CodeOcean::Config.new(:code_ocean).read.dig(:content_moderation, :report_emails).present?
```

Comment:
There is an outdated but still relevant conversation about memorizing this: https://github.com/openHPI/codeocean/pull/2946#discussion_r2143350685
Please check out how it is done in `CodeharborLinkPolicy`